### PR TITLE
fix: #432 Error when logout in the command-line tool

### PR DESCRIPTION
**Pull request by xquanluu**
#432 Error when logout in the command-line tool

### CODE REVIEW

**Comment by xquanluu** [comment]
I have read the CLA Document and I hereby sign the CLA

**Comment by efraa** [comment]
@xquanluu Thank you for solving this issue. I'd just recommend removing the `grpc-tools` version changes as it's not related to the issue.

File: package.json [hunk 1]
```diff
@@ -45,7 +45,7 @@
     "eslint-plugin-notice": "^0.9.10",
     "eslint-plugin-prettier": "^3.3.1",
     "grpc_tools_node_protoc_ts": "5.2.2",
-    "grpc-tools": "1.11.1",
```
Comment (efraa) [inline package.json]: @xquanluu Thank you for solving this issue. I'd just recommend removing the `grpc-tools` version changes as it's not related to the issue.

**Comment by xquanluu** [comment]
Hi @efraa 

I'm quite insterested in fonoster, do you have any other good first issue to me. Thanks a lot.

**Review by efraa — CHANGES_REQUESTED** [review]
Please remove the `package-lock.json` changes.

**Comment by psanders** [comment]
> Hi @efraa
> 
> I'm quite insterested in fonoster, do you have any other good first issue to me. Thanks a lot.

For sure! Be in the look out because we will continue to add more.

File: package.json (hunk 1 above)
Comment (xquanluu) [reply package.json]: yes, I fixed it, did change this version for to be able to build on Apple M1 chip.

For package-lock.json, it's automatically generated by npm run make. and it's keep changing every time, do you have any hint so I can avoid make changes in this file.

**Review by psanders — COMMENTED** [review]
Looks great.

File: package.json (hunk 1 above)
Comment (psanders) [reply package.json]: @xquanluu the `npm run make` it's meant to be run once. Most of the time using `run build` works fine.

**Review by psanders — APPROVED** [review]
All set here.

**Review by psanders — APPROVED** [review]
Reviewed.

**Review by psanders — APPROVED** [review]

**Review by psanders — APPROVED** [review]

**Review by efraa — APPROVED** [review]

**Comment by psanders** [comment]
@xquanluu thanks a lot for the PR. It helps a lot. Keep them coming 💪

**Pull request merged**